src/quacc/utils/files.py

```diff
@@ -69,9 +69,57 @@ def copy_decompress_files(
         if f_path.is_file():
             copy(f_path, Path(destination, f_path.name))
             decompress_file(Path(destination, f_path.name))
+        elif f_path.is_dir():
+            copy_decompress_files_from_dir(f_path, destination)
```

Comment:
I didn't comment on this previously, but this behavior (while essential for Espresso) will probably introduce confusion to some users/developers of other codes down the road. I imagine the default thought process for most people will be that if they provide a directory, that the directory will be copied (e.g. `cp -r` style) and then any files inside unzipped (`gunzip *`), whereas here what is really happening is a `cp *` on all the files _inside_ the directory. I will probably add a keyword argument at some point to more clearly address this.